### DISCUSSION TITLE
checking if worth doing? egpu

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2344] [Q&A] [answered]
i have a 64gb mac now ( got more ram ) and a slow amd 580 card that takes 15-20 mins a image

a guy at work says has a AORUS GTX 1080 GAMING BOX, doesn't use it no more, 12gb ram, and is thunderbolt 3, 

says is     NVIDIA and CUDA, DirectX 12 Ultimate, OpenGL 4.6, Vulkan

worth putting on the mac to spare monitor?

**Comment by mashb1t** [comment, marked as answer]
i'd personally advise against it, as Mac with CPU is most likely more powerful or even with using a GTX 1080. Also see the comparison in the [mininal requirements](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement).

**Comment by lgwjames** [comment]
o yes, i didn't see, so i would need something like a rtx 4XXX something
